docs/limitations.md

```diff
@@ -31,6 +31,11 @@ Therefore, when multiple pods requesting TopoLVM volumes are created at once, th
 Note that pod scheduling is also affected by the amount of CPU and memory.
 Because of this, this problem may not be observable.
 
+Snapshots should be created only for a `BOUND` PVC
```

Comment:
- does k8s by default forbids creating snaps for unbound PVCs?
- iirc, we are handling this case, isn't it?

Reply:
If a PVC is not Bound, no logical volume will be created in the backend. If a Snapshot creation is attempted for a `UNBOUND` PVC, it will fail with `source volume not found` in the backend (as even parent logical volume is not present)